docs/jwst/data_products/msa_metadata.rst

```diff
@@ -0,0 +1,221 @@
+.. _msa_metadata:
+
+MSA Meta Data File: ``msa``
```

Comment:
here and throughout: this is a nitpick, but to me it feels much more natural to use/see "metadata" without a space in between.  For example, [the Wikipedia article on the metadata](https://en.wikipedia.org/wiki/Metadata) does not use a space